### DISCUSSION TITLE
Grundsätzliche Fragen

### DISCUSSION THREAD

**Discussion by KeenBarete** [camueller/SmartApplianceEnabler#506] [Allgemein]
Hallo,
ich habe gerade dieses Projekt gefunden und finde es sehr interessant! In der Beschreibung steht das ein Sunny Home Manager gebraucht wird, meine frage wäre, geht es auch ohne wenn ein SMA Wechselrichter vorhaben ist? Oder würde es mit einem SMA Energy Meter funktionieren?

**Comment by camueller** [comment]
Es geht nicht ohne Sunny Home Manager.

**Comment by KeenBarete** [comment]
Hatte ich mir fast gedacht.
Ich möchte den Grundsätzlichen Verbrauch vom Haus inklusive meinen Verbrauch meiner "alten" 3 Phasigen Wärmepumpe anzeigen lassen. Gibt es da Modbuszähler welche besonders leicht einzubinden wären? Wie erfolgt die Kommunikation mit dem Home Manager? Ein zweites Netzwerkkabel?
Ich würde den SmartApplianceEnabler im Docker einrichten wollen.

**Comment by camueller** [comment]
Für Modbus brauchst Du einen USB-RS485-Adapter - siehe https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/Modbus_DE.md#modbusrtu

Auf den (ist eine normale serielle Schnittstelle) muss der SAE aus dem Docker-Container zugreifen können.

Ich selbst nutze diesen Zähler https://xn--stromzhler-v5a.eu/stromzaehler/drehstromzaehler/fuer-hutschiene-geeicht/246/sdm72dm-v2-mid-3-phasen-zweirichtungs-drehstromzaehler-mit-rs485-und-s0?c=94

Der Home-Manager muss sich im gleichen Netz befinden, wie der SAE.

**Comment by KeenBarete** [comment]
Ein USB-RS485-Adapter? Wäre ein RJ45-RS485-Adapter nicht besser?
Brauchen diese Adapter und evtl Zähler eine extra Stromzufuhr?
Brauche ich dann mehrere Ethernetkabel oder gar einen Switch?
